### DISCUSSION TITLE
Debug output

### DISCUSSION THREAD

**Discussion by kevinbeck76** [Dahkenangnon/flutter_feathersjs.dart#6] [General]
The library currently outputs a message each time a real-time update is delivered.  They look like:

service created
service object output

This is useful in a debug setting but in a situation with either frequent or large real-time updates the debug output is overwhelming.

Could some of the debug output be reduced?

For example in scketio_client.dart in the listen() function.

      print("$serviceName patched");
      print(patchedData);

Thanks for such a useful library.

Kevin

**Comment by Dahkenangnon** [comment]
> The library currently outputs a message each time a real-time update is delivered. They look like:
> 
> service created
> service object output
> 
> This is useful in a debug setting but in a situation with either frequent or large real-time updates the debug output is overwhelming.
> 
> Could some of the debug output be reduced?
> 
> For example in scketio_client.dart in the listen() function.
> 
> ```
>   print("$serviceName patched");
>   print(patchedData);
> ```
> 
> Thanks for such a useful library.
> 
> Kevin

Hi @kevinbeck76 

This is fixed.

**Reply by Dahkenangnon** [reply]
https://github.com/Dahkenangnon/flutter_feathersjs.dart/releases/tag/V1.0.10

**Reply by kevinbeck76** [reply]
Thanks for the quick fix.

**Reply by Dahkenangnon** [reply]
With pleasure.

The next major version will improve many thing; And it's for very soon.

**Comment by Dahkenangnon** [comment]
Fixed: https://pub.dev/packages/flutter_feathersjs
